inputs/LiMetal_PorousSep_oxygen.yaml

```diff
@@ -26,24 +27,24 @@ cell-description:
     thickness: 20e-6  # separator thickness, m
     n_points: 5       # Number of finite volumes to discretize
     electrolyte-phase: 'electrolyte' # Cantera phase name
-    sigma_io: 1.3 # S/m DOI:10.1149/2.0571912jes for 1M LiPF6 in EC:EMC (3:7 w:w) at 50 deg C
-    phi_0: 2.96 # Initial electric potential, V
+    sigma_io: 0.272 # S/m for 1M LiTFSI in TEGDME at 25 deg C // Chen, 2019
+    phi_0: 2.91 # Initial electric potential, V // Yin, 2017
     eps_electrolyte: 0.65 # Electrolyte volume fraction, -
   cathode:
     class: 'nucleation_single_particle'
-    bulk-phase: 'cathode' # Cantera phase name
-    surf-phase: 'cathode_electrolyte' # Cantera phase name
+    gas-phase: 'oxygen'
+    elyte-iphase: 'air-elyte-interface'
     electrolyte-phase: 'electrolyte' # Cantera phase name
-    conductor-phase: 'electron' # Cantera phase name
-    stored-ion: # Details on the stored species. Used for capacity calc.
-      name: Li2O2[cathode]
-      charge: 2
+    bulk-phase: 'conductor' # Cantera phase name
+    surf-iphase: 'cathode_surf' # Cantera phase name
```

Comment:
Does the `i` in `iphase` correspond to "interface," here and in `catalyst-iphase`?  If so, that is really already handled by "surf" being in the name.  I'd suggest something like:
`bulk-surf-phase: 'conductor_surf'`
(make the surface phase name match with the bulk phase corresponds to; you'll need to change the name in the Cantera portion of the input, as well) and
`catalyst-surf-phase: 'catalyst_surf'`